### ISSUE TITLE
user management is well broken : csv import deletes users, csv update is bugged, user update  broken

### ISSUE THREAD

**Issue by CPASimUSante** [claroline/Distribution#1419]
| Q             | A
| ------------- | ---
| Bug report?   | yes
| Version?      | master

- [ ]  In a new installation (i tested also on a travis instance), i created a admin user. Then as the admin (/admin/users/import) I import a list of users :

- i have a list of 5 users, 
- i choose "Creation only" (no other options checked)
- and click OK

=> my admin is overwritten with the last user info and all the other user are not created!

And a also tried the "create and update" option : i have also a error : 
`Notice: Undefined offset: 5`  : 

```
in vendor/claroline/distribution/main/core/Controller/Administration/UsersController.php at line 316   -
                        $datas = str_getcsv($line, ';');
                        $username = $datas[2];
                        $email = $datas[4];
                        $code = $datas[5];
                        $existingUser = $this->userManager->getUserByUsernameOrMailOrCode(
                            $username,
                            $email,
```
it seems the code is mandatory where it shouldn't

- [ ]  the command seems broken also : 
`php app/console claroline:users:load`  and i set the path.
It seems to do the same `XXX (XXX) being updated`
but not created

- [ ] the edit seems also problematic : (profile/profile/edit) i change first name, last name, username, administrative code and i have :
```
Catchable Fatal Error: Argument 1 passed to Claroline\CoreBundle\Manager\RoleManager::renameUserRole() must be an instance of Claroline\CoreBundle\Entity\Role, null given, called in /home/user/www/claro-master/Claroline/vendor/claroline/distribution/main/core/Controller/ProfileController.php on line 281 and defined
500 Internal Server Error - ContextErrorException
```

**Comment by ngodfraind** [comment]
you're right, it comes from the cod that shouldn't be mandatory. It's an easy fix.

**Comment by ngodfraind** [comment]
Fixed by #1421
